otterspace-xyz/badges-subgraph
# [DEV-522] - Revocation

In this PR we add support for revocation and reinstatement. 

Due to the potential of other developers using our subgraph on staging, for testing I deployed to a new Goerli subgraph, located at: https://thegraph.com/hosted-service/subgraph/grp06/badges-tester

To test:
- create a new badge spec
- add a wallet to the allow list
- claim the badge with that wallet
- query the subgraph and pull down the `badgeId` that you just minted
- query the subgraph to get the `raftTokenId` for that spec
- go to Etherscan and revoke the badge
- check the subgraph to see that the badge was revoked
- go to Etherscan and reinstate the badge
- check the subgraph to see that the badge is reinstated
